merge_requests.go

```diff
@@ -111,6 +111,8 @@ type ListMergeRequestsOptions struct {
 	State   *string `url:"state,omitempty" json:"state,omitempty"`
 	OrderBy *string `url:"order_by,omitempty" json:"order_by,omitempty"`
 	Sort    *string `url:"sort,omitempty" json:"sort,omitempty"`
+	Search  *string `url:"search,omitempty" json:"search,omitempty"`
+	Labels  Labels  `url:"labels,comma,omitempty" json:"labels,omitempty"`
```

Comment:
Again this field is only supported by the V4 API, so you should really rebase and point this PR to the `f-api-v4` branch.